demo.py

```diff
@@ -51,7 +56,7 @@ def main():
         "product_matchrules": [
             ["CREATE", "foo.tar.gz"],
         ],
-        "pubkeys": [bob_key["keyid"]],
+        "pubkeys": [bob_public["keyid"]],
```

Comment:
Wouldn't it be worth using our models to do this instead? so we don't have to perform these little changes every time.